### DISCUSSION TITLE
Error while creating NAT gateway route

### DISCUSSION THREAD

**Discussion by Annepuvarunkumar** [learndevopsonline/learndevopsonline#1674] [Q&A] [answered]
![Screenshot 2023-12-13 191251](https://github.com/learndevopsonline/learndevopsonline/assets/139056909/625bc295-b5f0-463f-bfed-ed6f0ef58e3b)

IP address: 3.91.47.121

**Comment by sreedharm07** [comment]
Can you once read the error 
It is expecting rtd-<here value> but its reciving another 
Please check variables you are passing to nat gate correctly

**Comment by sreedharm07** [comment, marked as answer]
Route table id you are giving is wrong check that and give properly

**Reply by Annepuvarunkumar** [reply]
yes i have passed subnet_ids, i recorrected it now its working fine. thank you
